src/react-components/misc/AppLogo.js

```diff
@@ -2,15 +2,22 @@ import React from "react";
 import PropTypes from "prop-types";
 
 import configs from "../../utils/configs";
-import { getAppLogo } from "../../utils/get-app-logo";
 import { ReactComponent as HmcLogo } from "../icons/HmcLogo.svg";
 import { isHmc } from "../../utils/isHmc";
+import { useDarkMode } from "../styles/theme";
 
 export function AppLogo({ className, forceConfigurableLogo }) {
+  const darkMode = useDarkMode();
   if (isHmc() && !forceConfigurableLogo) {
     return <HmcLogo className="hmc-logo" />;
   } else {
-    return <img className={className} alt={configs.translation("app-name")} src={getAppLogo()} />;
+    return (
+      <img
+        className={className}
+        alt={configs.translation("app-name")}
+        src={(darkMode && configs.image("logo_dark")) || configs.image("logo")}
```

Comment:
This isn't fully equivalent to the logic that was in `getAppLogo`. In particular, if your system theme is set to "light", but your Hubs theme is set to "Hubs Dark Mode" in our preferences panel, this will still use the "light" logo according to the system theme.
I think we still care about overriding the system theme with our theme preference, so we should account for that scenario.

Reply:
Thanks for catching that. I've updated.